src/types/gecko-profile.js

```diff
@@ -114,6 +115,10 @@ export type GeckoSampleStructWithResponsiveness = {|
   stack: Array<null | IndexIntoGeckoStackTable>,
   time: Milliseconds[],
   responsiveness: Array<?Milliseconds>,
+  // New values related to cpu usage.
+  threadCPUUsage?: Array<number | null>,
```

Comment:
I think `threadCPUDelta` would be a better name, to make it clearer that it's the change since the previous sample, and not a percentage.